icecreamwars/icecreamwars/src/Components/App/App.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./App.css";
+import Header from "../Header/Header";
+import AdDesigner from "../AdDesigner/AdDesigner";
+import Votes from "../Votes/Votes";
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <AdDesigner />
```

Comment:
I like to handle some of the major layout configuration here in App where components are being imported. You can wrap AdDesigner and Votes in one div that is responsive to screen size like this
```
    <div className="App">
      <Header />
      <div className="content-display">
        <AdDesigner />
        <Votes />
      </div>
    </div>
```

Then you can assign flex directions in the CSS file like this: 
```.content-display {
  display: flex;
  flex-direction: column;
}

@media screen and (min-width: 768px) {
  .content-display {
    flex-direction: row;
    justify-content: space-between;
  }
}
```